packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -30,6 +38,12 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
 
   private vmcDeviceId?: string;
 
+  private announceStatus: string;
+
+  private captionStatus: string;
+
+  private turnOnCaptionsWaitingOnlinePromise: Promise<void> | null;
```

Comment:
question: What does turnOnCaptionsWaitingOnlinePromise do? I can't see it being used anywhere

Reply:
Removed.
Thanks!